### PR TITLE
[bitnami/vault] Enabling Vault run in one single namespace

### PR DESCRIPTION
**Pull request by matheusmgon**
continue PR: https://github.com/bitnami/charts/pull/25328

Feature to enable Vault run in one single Namespace

### CODE REVIEW

**Comment by javsalgar** [comment]
Hi!

Thank you so much for the PR! Could you fix the DCO issue?

**Comment by matheusmgon** [comment]
> Hi!
> 
> Thank you so much for the PR! Could you fix the DCO issue?

Now, everything is ok

File: bitnami/vault/values.yaml [hunk 1]
```diff
@@ -1760,6 +1760,17 @@ injector:
...
+  webhook: {}
+  ## @param injector.webhook Enabling specify which namespace you want to work with webhook
+  ## Example:
+  ##   namespaceSelector:
+  ##     matchExpressions:
+  ##     - key: kubernetes.io/metadata.name
+  ##       operator: In
+  ##       values:
+  ##         - "{{ .Release.Namespace }}"
+  #namespaceSelector: {}
```
Comment (migruiz4) [inline bitnami/vault/values.yaml:1773]: I will apply this small change to help improve the values.yaml readability.
```suggestion
  ## @param injector.webhook.namespaceSelector Enabling specify which namespace you want to work with webhook
  ## Example:
  ##   namespaceSelector:
  ##     matchExpressions:
  ##     - key: kubernetes.io/metadata.name
  ##       operator: In
  ##       values:
  ##         - "{{ .Release.Namespace }}"
  webhook:
    namespaceSelector: {}
```

**Review by migruiz4 — APPROVED** [review]
Looks good to me, thank you for applying all the suggested changes!

**Pull request merged**